docs/authentication/social-connections/google.mdx

```diff
@@ -39,15 +39,15 @@ Optionally, you can also [set additional OAuth scopes that your application migh
 
 As a quick recap, if you followed this guide, you should have done the following:
 
-- created an OAuth client ID in Google Cloud Platform console
-- added the authorized redirect URI in that OAuth client ID, as provided by Clerk Dashboard
-- set the proper Client ID and Client secret values in Clerk Dashboard, as provided by the Google Cloud Platform console
+- Created an OAuth client ID in Google Cloud Platform console
+- Added the authorized redirect URI in that OAuth client ID, as provided by Clerk Dashboard
+- Set the proper Client ID and Client secret values in Clerk Dashboard, as provided by the Google Cloud Platform console
 
-In the Clerk Dashboard configuration screen, you should end up with something like this:
+In the Clerk Dashboard, you should end up with something like this:
 
-![Client ID and Client secret filled in. Authorized redirect URI copied and entered into Google Cloud Platform console.](/docs/images/authentication-providers/google/8bf67be45ecccaf962a881fb09a61ed810e818cf-563x869.png)
+![Client ID and Client secret filled in. Authorized redirect URI copied and entered into Google Cloud Platform console.](/docs/images/authentication-providers/google/configuration.png)
 
-Finally, click **Save** so that the settings are applied. Congratulations! Social connection with Google is now configured for your instance.
+Finally, click **Add connection** so that the settings are applied. Congratulations! Social connection with Google is now configured for your instance.
```

Comment:
```suggestion
Finally, select **Add connection** so that the settings are applied. Congratulations! Social connection with Google is now configured for your instance.
```